Quaver.Shared/Screens/Gameplay/Rulesets/Keys/Playfield/GameplayPlayfieldKeysStage.cs

```diff
@@ -550,11 +554,22 @@ private void CreateJudgementHitBurst()
             // Set size w/ scaling.
             var size = new Vector2(firstFrame.Width, firstFrame.Height) * Skin.JudgementHitBurstScale / firstFrame.Height;
 
-            JudgementHitBurst = new JudgementHitBurst(Screen, frames, size, Skin.JudgementBurstPosY)
+            var judgementBurstCount = skin.DisplayJudgementsInEachColumn == true ? Screen.Map.GetKeyCount(Screen.Map.HasScratchKey) : 1;
+
+            for (var i = 0; i < judgementBurstCount; i++)
             {
-                Parent = Playfield.ForegroundContainer,
-                Alignment = Alignment.MidCenter,
-            };
+                var judgementHitBurst = new JudgementHitBurst(Screen, frames, size, Skin.JudgementBurstPosY)
+                {
+                    Parent = Playfield.ForegroundContainer,
+                    Alignment = Alignment.MidCenter,
+                    X = skin.DisplayJudgementsInEachColumn == true ? Receptors[i].X - ((Playfield.Width / 2) - (Playfield.LaneSize / 2)) : 0
```

Comment:
Remove `== true` and extract `Playfield.Width / 2) - (Playfield.LaneSize / 2)` as a variable